addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js

```diff
@@ -194,13 +196,18 @@ export class Powerbox {
         this.render(this._currentOpenOptions.commands);
 
         let initialTarget;
+        const currentElement = this._currentOpenOptions.currentElement || this.options.editable;
 
         const keyup = async ev => {
             if (ev.key === 'ArrowDown' || ev.key === 'ArrowUp') {
                 ev.preventDefault();
                 return;
             }
             if (!initialTarget) return;
+            // Update initialTarget in case of backspace.
+            if (!currentElement.contains(initialTarget)) {
+                initialTarget = selection.anchorNode;
+            }
```

Comment:
Same questions as in the 16 branch: why is this needed, and why specifically for backspace?

Reply:
For backspace because on backspace the `initialTarget` which is `selection.anchorNode` is removed and it is no longer in `currentElement` so I reset it. In general case where `selection.anchorNode` is no longer present in `currentElement`.